test/modelmanagers/tagModelManager.test.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai');
+const { findOrCreateTag, getTag } = require('../../lib/modelManagers/tagModel');
+
+describe('create article tag', () => {
+  const tag = 'JavaScript';
+  it('should create a tag in the tag table', async () => {
+    const createdTag = await findOrCreateTag(tag);
+    expect(createdTag.tagName).to.equal('JavaScript');
+  });
+
+  it('should respond with a tag object when a valid tagName is passed', async () => {
+    const createdTag = await findOrCreateTag(tag);
+    expect(createdTag).to.contain(Object);
+  });
+
+  it('should get the tags in the tag table', async () => {
+    const foundTag = await getTag(tag);
+    expect(foundTag[0]).to.contain(Object);
```

Comment:
What about tests for the object properties?

Reply:
I tested for this below. kindly check

Reply:
Seen